.prettierrc

```
@@ -0,0 +1,7 @@
+{
+	"useTabs": true,
```

Acho que é válido manter os espaços
```suggestion
	"useTabs": false,
```

[spaces > tabs](https://www.youtube.com/watch?v=SsoOG6ZeyUI)